File: modules/AcroStack.AppUsers/Dtos/AppUserDto.cs
using System;
using Volo.Abp.Application.Dtos;

namespace AcroStack.AppUsers;

public class AppUserDto : EntityDto<Guid>
{
    public string UserName { get; set; } = null!;
    public string? Email { get; set; }
    public string? Name { get; set; }
    public string? Surname { get; set; }
    public string? PhoneNumber { get; set; }
    public bool IsActive { get; set; }
    public DateTime CreationTime { get; set; }
}

File: react/src/api/models/acroStack/appUsers/AppUserDto.ts
/* oxlint-disable */

export type AcroStackAppUsersAppUserDto = {
    /**
     * @description
     * Format: `uuid`
     * @type string | undefined
    */
    id?: string;
    userName?: string | null;
    email?: string | null;
    name?: string | null;
    surname?: string | null;
    phoneNumber?: string | null;
    isActive?: boolean;
    /**
     * @description
     * Format: `date-time`
     * @type string | undefined
    */
    creationTime?: string;
};
